example/samples/install_coreos_payload_full.json

```diff
@@ -0,0 +1,43 @@
+{
+    "options": {
+        "defaults": {
+            "hostname": "Coreos-node",
+            "installDisk": "/dev/sda",
+            "comport": "ttys0",
+            "repo": "http://172.31.128.1:9080/coreos/amd64-usr",
```

Comment:
I am not sure if change the host to a render-able ```{{ file.server }}/coreos/adm64-usr``` is a good idea. ignore me if it's not. @yyscamper